### PR TITLE
Edit pass on Chapter 01

### PR DESCRIPTION
**Pull request by rupsis**
All changes presented here are _suggestions_. 

These suggestions are largely subjective, with the occasional objective suggestion (i.e spelling correction). 
I'll try to annotate my changes, and provide additional editing notes. 

Feel free to either open a dialogue on this PR, and I'll work with you on edits. Or use this as a reference (cherry pick suggestions) and close the PR out. 

**Note:** This is an editing pass, and by no means a comprehensive revision.

### CODE REVIEW

File: book/MovingToTheGPU.html [hunk 1]
```diff
@@ -10,52 +10,52 @@
 
 Introduction
 ====================================================================================================
-_Ray Tracing_ is a rendering method that simulates the flow of light. It can faithfully recreate a
-variety of optical phenomena and can be used to render photorealistic images. It is one of the
-foundational techniques in Computer Graphics used to model light transport.
+_Ray Tracing_ is a rendering method in Computer Graphics that simulates the flow of light.
+It can accurately recreate a variety of optical phenomena and be used to render photorealistic images.
```
Comment (rupsis) [inline book/MovingToTheGPU.html:14]: Slightly more concise opening

File: book/MovingToTheGPU.html [hunk 2]
```diff
@@ -10,52 +10,52 @@
...
 
-_Path tracing_ is an application of this approach in order to compute _Global Illumination_. Its
+_Path tracing_ is an application of this approach, which is used to compute _Global Illumination_. Its
 core idea is to repeatedly trace millions of random rays through the scene and bounce them off
-objects based on surface properties. The algorithm is remarkably simple and it is relatively easy
-to build a basic path tracer that supports a small number of material and geometry types. Peter
-Shirley's [_Ray Tracing In One Weekend_][RTIOW] shows you how to get started with a CPU path tracer
+objects based on surface properties. The algorithm is remarkably simple and relatively easy
+to build a basic path tracer, which supports a small number of material and geometry types. Peter
+Shirley's [_Ray Tracing In One Weekend_][RTIOW] (RTIOW) shows you how to get started with a CPU path tracer
```
Comment (rupsis) [inline book/MovingToTheGPU.html:20]: The acronym "RTIOW" is used without being introduced.

File: book/MovingToTheGPU.html [hunk 3]
```diff
@@ -10,52 +10,52 @@
...
-A big challenge with path tracing is its high computational cost. Rendering a complex scene takes a
-long time and this get worse as complexity increases. This has historically made path
+A challenge with path tracing is its high computational cost. Rendering a complex scene takes a
+long time and grows worse as complexity increases. This has historically made path
 tracing unsuitable for real-time applications. Fortunately -- like many problems in Computer
-Graphics -- the algorithm lends itself very well to parallelism and it is possible to achieve a
-significant speedup by distributing the work across many processor cores. This brings us to GPUs.
+Graphics -- the algorithm lends itself very well to parallelism. It is possible to achieve a
+significant speedup by distributing the work across many processor cores. This brings us to
+the Graphics Processing Unit (GPU).
```
Comment (rupsis) [inline book/MovingToTheGPU.html:28]: GPU acronym was used (albeit just slightly) before it was introduced.

File: book/MovingToTheGPU.html [hunk 4]
```diff
@@ -10,52 +10,52 @@
...
 
-The GPU (Graphics Processing Unit) is a type of processor designed to run the same set of oparations
+The GPU is a type of processor designed to run the same set of operations
 over large amounts of data in parallel. This parallelism has been instrumental to achieving
 realistic visuals in real-time applications like video games. GPUs were traditionally used to
-accelerate _scanline rasterization_ but they have since become programmable and capable of running
-a variety of parallel workloads. Notably, dedicated hardware cores to accelerate ray tracing are
-becoming more common in modern GPUs.
```
Comment (rupsis) [inline book/MovingToTheGPU.html:35]: Sentence read a little awkward.

File: book/MovingToTheGPU.html [hunk 5]
```diff
@@ -10,52 +10,52 @@
...
 avoid most optimizations in favor of simplicity but the renderer will be able to achieve interactive
 frame rates on a decent GPU.[^ch1]
 
-The book follows a similar progression to [_Ray Tracing In One Weekend_][RTIOW]. I cover some of
+This book follows a similar progression to [_Ray Tracing In One Weekend_][RTIOW]. I cover some of
 the same material but I highly recommend completing _RTIOW_ before embarking on building
-the GPU version since doing so will teach you the path tracing algorithm in a much more approachable
+the GPU version. Doing so will teach you the path tracing algorithm in a much more approachable
 way and it will make you appreciate both the advantages and challenges of moving to a GPU-based
 architecture.
```
Comment (rupsis) [inline book/MovingToTheGPU.html:55]: A rather long run on sentence. Should we use the RTIOW acronym over full title for consistency?

File: book/MovingToTheGPU.html [hunk 6]
```diff
@@ -113,31 +113,30 @@
...
 
 * For ease of Rust development I use [*anyhow*](https://docs.rs/anyhow/latest/anyhow/) and
   [*bytemuck*](https://docs.rs/bytemuck/latest/bytemuck/). *anyhow* is a popular error handling
   utility and integrates seamlessly. *bytemuck* provides a safe abstraction for the equivalent of
-  `reinterpret_cast` in C++, which normally requires [`unsafe`][rust-unsafe] Rust. I use it to
+  `reinterpret_cast` in C++, which normally requires [`unsafe`][rust-unsafe] Rust. It's used to
   bridge CPU data types with their GPU equivalents.
 
 * Lastly, I use [*pollster*](https://docs.rs/pollster/latest/pollster/) to execute asynchronous
   wgpu API functions (which is only called from a single line).
```
Comment (rupsis) [inline book/MovingToTheGPU.html:126]: I removed some redundant "I use" verbiage, but we could go further and initially state "These are the dependencies used". Each dependency listed then can have justification for it's use.

File: book/MovingToTheGPU.html [hunk 7]
```diff
@@ -113,31 +113,30 @@
...
+as such my GPU code examples are written in *WebGPU Shading Language* (WGSL)[^ch1.2.1].
 <!-- Have GLSL examples too? -->
 
-The [wgpu API documentation](https://docs.rs/wgpu/latest/wgpu/) and the
-[WebGPU](https://www.w3.org/TR/webgpu/) and [WGSL](https://www.w3.org/TR/WGSL/) specifications are
-great references to keep handy when you're
-developing.
+The [wgpu API documentation](https://docs.rs/wgpu/latest/wgpu/), 
+[WebGPU](https://www.w3.org/TR/webgpu/), and [WGSL](https://www.w3.org/TR/WGSL/) specifications are
+great references to keep handy when you're developing.
```
Comment (rupsis) [inline book/MovingToTheGPU.html:139]: In line list _probably_ should be a bulleted list, but 🤷

File: book/MovingToTheGPU.html [hunk 8]
```diff
@@ -10,52 +10,52 @@
...
+long time and grows worse as complexity increases. This has historically made path
 tracing unsuitable for real-time applications. Fortunately -- like many problems in Computer
-Graphics -- the algorithm lends itself very well to parallelism and it is possible to achieve a
-significant speedup by distributing the work across many processor cores. This brings us to GPUs.
+Graphics -- the algorithm lends itself very well to parallelism. It is possible to achieve a
+significant speedup by distributing the work across many processor cores. This brings us to
+the Graphics Processing Unit (GPU).
 
-The GPU (Graphics Processing Unit) is a type of processor designed to run the same set of oparations
+The GPU is a type of processor designed to run the same set of operations
```
Comment (rupsis) [inline book/MovingToTheGPU.html:30]: Spelling -> operations

File: book/MovingToTheGPU.html (hunk 2 above)
Comment (hollasch) [reply book/MovingToTheGPU.html:20]: In Markdeep, the text inside the square brackets is the link text. Here, "_Ray Tracing in One Weekend_".

File: book/MovingToTheGPU.html (hunk 6 above)
Comment (armansito) [reply book/MovingToTheGPU.html:126]: I like the passive voice that you went with.

File: book/MovingToTheGPU.html (hunk 5 above)
Comment (armansito) [reply book/MovingToTheGPU.html:55]: Sounds good!

**Comment by armansito** [comment]
Hey @rupsis, sorry for taking a long time on getting back to you. I agree with most of your suggestions, I went ahead and applied them (with some minor tweaks) to the dev branch.

Please go ahead and close this PR. Thank you very much for the edits!

**Comment by rupsis** [comment]
> Hey @rupsis, sorry for taking a long time on getting back to you. I agree with most of your suggestions, I went ahead and applied them (with some minor tweaks) to the dev branch.
> 
> Please go ahead and close this PR. Thank you very much for the edits!

Hey @armansito No worries at all :) 

I'm happy to provide more edits in the future (provided they're useful) if you're interested.

**Comment by armansito** [comment]
> > Hey @rupsis, sorry for taking a long time on getting back to you. I agree with most of your suggestions, I went ahead and applied them (with some minor tweaks) to the dev branch.
> > Please go ahead and close this PR. Thank you very much for the edits!
> 
> Hey @armansito No worries at all :)
> 
> I'm happy to provide more edits in the future (provided they're useful) if you're interested.

Yes, I'd definitely appreciate your feedback on the rest of the book, especially once it's complete.

**Pull request closed without merging**